### ISSUE TITLE
Refiner: what is allowed in FASTA header lines?

### ISSUE THREAD

**Issue by mcfrith** [Dfam-consortium/RepeatModeler#185]
For Refiner in RepeatModeler-2.0.3:
I assume that the FASTA header lines can contain anything I want, and it won't make any difference.

But when I happened to use headers line this:
`>aERV-int.8 chr8`

I got this message:
`Could not parse identifier: chr8`

I think this is either a code bug or a documentation bug.

**Comment by rmhubley** [comment]
You are correct this is a documentation issue.  Refiner was designed as in internal component of RepeatModeler and hasn't received as much user-interface attention as the other programs.  The FASTA description field is reserved for RepeatModeler use at this time.  As Refiner is getting more attention we will work to remove this restriction in future releases.
